### PR TITLE
issue535 prevent negative solutions for radiative temperature

### PR DESCRIPTION
**Pull request by PMehrfeld**
Introduce max(T,100) to prevent negative solutions for the temperature
Related to #535.

### CODE REVIEW

File: AixLib/Utilities/HeatTransfer/HeatToStar.mo [hunk 1]
```diff
@@ -6,7 +6,12 @@ model HeatToStar "Adaptor for approximative longwave radiation exchange"
   parameter Modelica.SIunits.Emissivity eps = 0.95 "Emissivity";
 equation
   Therm.Q_flow + Star.Q_flow = 0;
-  Therm.Q_flow = Modelica.Constants.sigma * eps * A * (Therm.T * Therm.T * Therm.T * Therm.T - Star.T * Star.T * Star.T * Star.T);
+  
+  // To prevent negative solutions for T, the max() expression is used.
+  // Negative solutions also occurs when using max(T,0), therefore, 100 K is used.
```
Comment (mlauster) [inline AixLib/Utilities/HeatTransfer/HeatToStar.mo]: Negative solutions also occur ...

File: AixLib/Utilities/HeatTransfer/HeatToStar.mo [hunk 2]
```diff
@@ -6,7 +6,12 @@ model HeatToStar "Adaptor for approximative longwave radiation exchange"
...
+  Therm.Q_flow = Modelica.Constants.sigma * eps * A * (
+    max(Therm.T,100) * max(Therm.T,100) * max(Therm.T,100) * max(Therm.T,100) -
```
Comment (mlauster) [inline AixLib/Utilities/HeatTransfer/HeatToStar.mo]: Would 1 K also work? Just to choose the most reasonable value...

File: AixLib/Utilities/HeatTransfer/HeatToStar.mo (hunk 1 above)
Comment (PMehrfeld) [reply AixLib/Utilities/HeatTransfer/HeatToStar.mo]: Thanks

File: AixLib/Utilities/HeatTransfer/HeatToStar.mo (hunk 2 above)
Comment (PMehrfeld) [reply AixLib/Utilities/HeatTransfer/HeatToStar.mo]: This works, too. Good proposal, I changed it in my last commit.

**Comment by mlauster** [comment]
@PMehrfeld, go ahead and merge

**Pull request merged**